test/Ocelot.AcceptanceTests/Steps.cs

```diff
@@ -73,29 +54,41 @@ public class Steps : IDisposable
         private HttpContent _postContent;
         private BearerToken _token;
         public string RequestIdKey = "OcRequestId";
-        private readonly Random _random;
+        private readonly Random _random = new();
+        private readonly string _ocelotConfigFileName = $"{Guid.NewGuid():N}-ocelot.json";
         private IWebHostBuilder _webHostBuilder;
         private WebHostBuilder _ocelotBuilder;
         private IWebHost _ocelotHost;
         private IOcelotConfigurationChangeTokenSource _changeToken;
-
-        public Steps()
-        {
-            _random = new Random();
-        }
+        private bool _disposedValue;
 
         public async Task ThenConfigShouldBe(FileConfiguration fileConfig)
         {
             var internalConfigCreator = _ocelotServer.Host.Services.GetService<IInternalConfigurationCreator>();
-
             var internalConfigRepo = _ocelotServer.Host.Services.GetService<IInternalConfigurationRepository>();
-            var internalConfig = internalConfigRepo.Get();
 
+            var internalConfig = internalConfigRepo.Get();
```

Comment:
Was it a root cause? I mean, absence of this call?

Reply:
It was already in there.